.storybook/manager.js

```diff
@@ -0,0 +1,2 @@
+import { addons } from "@storybook/addons";
+addons.setConfig({ showRoots: false });

```

Comment:
I'm surprised the linter does not yell at us for using semicolons at the end of the line. it's something we can look at in a separate PR